### PR TITLE
Remove the Webhook setting of cert-manager

### PR DESCRIPTION
**Pull request by meyskens**
# Remove the Webhook settiing of cert-manager

This removes the webhook flag from the cert-manager component as it did not function any longer. The no webhook version no longer exists.

# How to use

- install cert-manager

# Testing done

- Checked creation of fresh cluster
- Checked when upgrading from 0.6.1 to this commit

# Notes

~~So removing the HCL flag might need some upgrade path, keeping it draft for now till I worked that out :) Just a quick patch in between other work.~~

As discussed in Slack removing the flag is fine pre-1.0 , should be released in a new minor release with a note however.

### CODE REVIEW

File: docs/configuration-reference/components/cert-manager.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ cert-manager component configuration example:
 component "cert-manager" {
   email = "example@example.com"
   namespace = "cert-manager"
```
Comment (invidian) [inline docs/configuration-reference/components/cert-manager.md:25]: Unrelated, but we could fix formatting here.
```suggestion
  email     = "example@example.com"
  namespace = "cert-manager"
```

File: docs/configuration-reference/components/cert-manager.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Cert-Manager configuration reference for Lokomotive
+title: cert-manager configuration reference for Lokomotive
```
Comment (invidian) [inline docs/configuration-reference/components/cert-manager.md:2]: Nit: this could go into separate commit.

**Review by invidian — APPROVED** [review]
Looks good, we could fix some nits along the way, but it's not a blocker to me.

File: docs/configuration-reference/components/cert-manager.md (hunk 1 above)
Comment (meyskens) [reply docs/configuration-reference/components/cert-manager.md]: Fixed!

File: docs/configuration-reference/components/cert-manager.md (hunk 2 above)
Comment (meyskens) [reply docs/configuration-reference/components/cert-manager.md:2]: Done!

**Review by invidian — APPROVED** [review]

File: docs/configuration-reference/components/cert-manager.md [hunk 3]
```diff
@@ -23,7 +23,6 @@ cert-manager component configuration example:
 component "cert-manager" {
```
Comment (surajssd) [inline docs/configuration-reference/components/cert-manager.md:23]: Nit: (being pedantic) the commit message is not wraped to 72 chars. The PR is good :+1: .

Comment (invidian) [reply docs/configuration-reference/components/cert-manager.md:23]: True. I can recommend configuring your editor to wrap the lines for you for Git commit messages. I think `vim` does that automagically.

Comment (meyskens) [reply docs/configuration-reference/components/cert-manager.md:23]: Changed my editor settings :) fixed now!

**Review by invidian — DISMISSED** [review]

**Review by ipochi — APPROVED** [review]
LGTM.

Lovely work @meyskens :tada:

**Comment by ipochi** [comment]
re-running the failed CI jobs.

**Review by surajssd — DISMISSED** [review]

**Comment by meyskens** [comment]
Rebased on master to have some CI fixes in the branch

**Review by invidian — APPROVED** [review]

**Review by iaguis — APPROVED** [review]
